doc/visual-programming/source/widgets/data/rank.rst

```diff
@@ -32,13 +32,25 @@ with the class.
 1. Select attributes from the data table.
 2. Data table with attributes (rows) and their scores by different
    scoring methods (columns)
-3. If '*Commit on any change*' ticked, the widget automatically
+3. Produce a report. 
+4. If '*Send Automatically*' is ticked, the widget automatically
    communicates changes to other widgets.
 
+Scoring methods
+---------------
+
+1. `Information Gain <https://en.wikipedia.org/wiki/Information_gain_in_decision_trees>`_: in machine learning, it is synonymous with `Kullback-Leibler divergence <https://en.wikipedia.org/wiki/Kullback%E2%80%93Leibler_divergence>`_ and in the context of decision trees it can be used interchangeably with the term '*mutual information*'
+2. `Gain Ratio <https://en.wikipedia.org/wiki/Information_gain_ratio>`_: a ratio of information gain to the intrinsic information
+3. `Gini <https://en.wikipedia.org/wiki/Gini_coefficient>`_: measures the inequality among values of a frequency distribution 
+4. `ANOVA <https://en.wikipedia.org/wiki/Analysis_of_variance>`_: a collection of statistical models used to analyze the differences among group means and their associated procedures
+5. `Chi2 <https://en.wikipedia.org/wiki/Chi-squared_distribution>`_: the distribution of a sum of the squares of k independent standard normal random variables
+6. `ReliefF <https://en.wikipedia.org/wiki/Relief_(feature_selection)>`_: a feature selection algorithm used in binary classification
+7. FCBF (Fast Correlation Based Filter)
+
```

Comment:
@janezd @BlazZupan I think we should make this somehow more understandable. I don't know if it's even possible, but it's worth a try...?